packages/xdl/src/Exp.ts

```diff
@@ -161,7 +162,7 @@ export async function extractTemplateAppAsync(
 }
 
 async function initGitRepoAsync(root: string) {
-  if (process.platform === 'darwin' && !Binaries.isXcodeInstalled()) {
+  if (process.platform === 'darwin' && !hasbin.sync('git')) {
```

Comment:
Thanks for fixing this issue!

I think we could remove this check and just try to call `git rev-parse` (see lines below). If that throws an error where `e.code` is `ENOENT` that means git is not installed and we can print a warning.

Reply:
Sounds good to me. I was wondering why we should check explicitly on darwin but not on e.g. Linux.

Reply:
I've updated the change. Please let me know if the way I'm checking for the ENOENT exception is OK.